src/deepseek_r1/r1.py

```
@@ -3,730 +3,378 @@
-                    apply_diff_edit(
-                        edit_info.path,
-                        edit_info.original_snippet,
-                        edit_info.new_snippet,
-                        getattr(edit_info, 'line_numbers', None)
-                    )
-            else:
-                console.print("[yellow]ℹ[/yellow] Skipped applying diff edits.", style="yellow")
+    def _handle_response(self, response: Dict):
+        """Process and display AI response"""
+        try:
+            validated = AssistantResponse(**response)
```

Ruff (0.8.2)

Undefined name `AssistantResponse`

(F821)